Demo/EarlGreyExample/README.md

```diff
@@ -16,3 +16,14 @@ Ensure that your version of CocoaPods is upgraded to the latest 1.x version sinc
 running `gem install cocoapods`.
 
 Run the tests that you want to see in action.
+
+## Getting Started - Carthage
+
+From the directory containing the example source, run:
+
+- `gem install earlgrey`
+- `earlgrey install --carthage --swift-version=3.0 --target=EarlGreyExampleSwiftTests`
+- `carthage update EarlGrey --platform ios`
+- `open EarlGreyExample.xcworkspace`
```

Comment:
How is `EarlGreyExample.xcworkspace` generated without CocoaPods? Does it already have to exist?